README.md

```diff
@@ -476,18 +481,50 @@ Anatomy of a Redis command
 
 All the Redis commands are defined in the following way:
 
-    void foobarCommand(client *c) {
-        printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
-        addReply(c,shared.ok); /* Reply something to the client. */
+```c
+void foobarCommand(client *c) {
+    printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
+    addReply(c,shared.ok); /* Reply something to the client. */
+}
+```
+
+There will be a `foobar.json` JSON file to record all the metadata for this command:
+
+```json
+{
+    "FOOBAR": {
+        "summary": "A foobar command.",
+        "complexity": "O(1)",
+        "group": "connection",
+        "since": "x.0.0",
+        "arity": 2,
+        "function": "foobarCommand",
+        "command_flags": [
+            "LOADING",
+            "FAST"
+        ],
+        "acl_categories": [
+            "CONNECTION"
+        ],
+        "arguments": [
+            {
+                "name": "message",
+                "type": "string"
+            }
+        ]
     }
+}
+```
 
-The command is then referenced inside `server.c` in the command table:
+The command is then referenced inside `commands.c`, see above file part for more details:
 
-    {"foobar",foobarCommand,2,"rtF",0,NULL,0,0,0,0,0},
+```c
+{"foobar","A foobar command.","O(1)","x.0.0",CMD_DOC_NONE,NULL,NULL,COMMAND_GROUP_CONNECTION,FOOBAR_History,FOOBAR_tips,foobarCommand,2,CMD_LOADING|CMD_FAST,ACL_CATEGORY_CONNECTION,.args=FOOBAR_Args},
+```
 
 In the above example `2` is the number of arguments the command takes,
-while `"rtF"` are the command flags, as documented in the command table
-top comment inside `server.c`.
+while `CMD_LOADING` and `CMD_FAST` are the command flags, as documented in the `redisCommand`
+top comment inside `server.h`. For more details, please refer to the `COMMAND` command.
```

Comment:
unlike before, where `2` was just an element in a struct initializer, now it's a field in a json file, with a name ("arity"), so i don't think we need to describe it.
let's just refer to the server.h comment that describes the flags, and also to https://redis.io/commands/command/